pom.xml

```diff
@@ -22,7 +22,7 @@
 
     <groupId>com.ubirch.util</groupId>
     <artifactId>ubirch-elasticsearch-utils_${scala.compat.version}</artifactId>
-    <version>0.2.8-SNAPSHOT</version>
+    <version>0.2.9-SNAPSHOT</version>
```

Comment:
I think it's nice to have a release tag with this version and a description of this change

Reply:
you mean to create a release with github as we did with the certifyAPI?